packages/terra-action-footer/src/ActionFooter.module.scss

```diff
@@ -1,14 +1,21 @@
 :local {
-  .action-footer {
+  .action-footer,
+  .action-footer-end {
     display: flex;
+    flex-flow: row wrap;
+    width: 100%;
   }
 
-  .empty-center {
-    flex: 1 1 auto;
+  .action-footer {
+    justify-content: space-between;
+
```

Comment:
would it make more sense to set `align-items: center` here instead of overriding the inner `start/end-actions` with `align-self: center`?

Reply:
I'm cool with this as-is.